submitted_models/coro_pam_sensor_config_1/specifications.md

```diff
@@ -0,0 +1,75 @@
+<!--- This is a Markdown description of a robot model submitted for inclusion in the
+DARPA Subterranean Challenge Technology Repository -->
+# Coordinated Robotics - Pam
+This specifications.md file is a description and proof of virtual model validation for
+Coordinated Robotics Pam with Sensor Configuration 1. This robot may be launched using
+a `ign launch` command with the variable name `coro_pam_sensor_config_1`.
+
+## Description
+Pam is a quadrotor UAV with 3 D435 cameras.  Pam weighs 3.6KG.
+
+## Usage Instructions
+The robot can be used in the same by sending Twist commands to the vehicle_name/cmd_vel ROS topic.
+
+
+## Usage Rights
+
+
+## Cost and Scale
+Pam has an estimated duplication cost of USD 2160. It weighs 3557 grams.
+Cost breakdown:
+$600  - 3 x D435
+$540 - Frame / motors / props / batteries / controllers
+$500 -  Computer - GB-BRi5-8250
+$300 - LIDAR-Lite V3HP x 2
+$220 - lights / connectors /switches / DC-DC / misc
+
+
+## Sensors
+Pam Sensor Config 1 uses:
+* 3 x D435 modeled by rgbd_camera plugin
+* 1 x Kakute F7 - modeled by air_pressure plugin, imu_sensor and magnetometer plugin
```

Comment:
Looking at the Kakute 57 specs, it does not have an air_pressure sensor

Reply:
On http://www.holybro.com/product/kakute-f7-v1-5/ there's a bullet that reads as follows:
> Ready for autonomous flight: Integrated BMP280 barometer and SCL/SDA pads for use with external GPS/magnetometer units.
> Specifications: * Barometer: BMP280
Regardless, barometer and magnetometer are standard on all UAVs in the SubT repository regardless of whether their real-world counterparts contain these sensors.